tests/EvmAcceptanceTests/artifacts/scilla.cache

```diff
@@ -0,0 +1 @@
+{"contracts/scilla/HelloWorld.scilla":{"hash":"d620d417615518045bd7099fe14ccd3c","path":"contracts/scilla/HelloWorld.scilla","parsedContract":{"name":"HelloWorld","transitions":[{"type":"CompTrans","name":"setHello","params":[{"type":"String","name":"msg"}]},{"type":"CompTrans","name":"getHello","params":[]}],"fields":[{"type":"String","name":"welcome_msg"}],"constructorParams":[{"type":"ByStr20","name":"owner"}]}},"contracts/scilla/SetGet.scilla":{"hash":"9f5952c6a835efe284afcebfbed953a6","path":"contracts/scilla/SetGet.scilla","parsedContract":{"name":"SetGet","transitions":[{"type":"CompTrans","name":"set","params":[{"type":"Uint128","name":"v"}]},{"type":"CompTrans","name":"emit","params":[]}],"fields":[{"type":"Uint128","name":"value"}],"constructorParams":null}}}
```

Comment:
Do we need this?

Reply:
It doesn't harm anyone. I did commit it purposefully. Why? Because if it exists in artifacts, it doesn't try to run scilla-fmt to generate one. so if someone doesn't have scilla-fmt everything works for him.